spec/hosts/nodes/tang01.dev.lsst.org_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'tang01.dev.lsst.org', :sitepp do
+  on_supported_os.each do |os, facts|
+    next unless os =~ %r{almalinux-9-x86_64}
+
+    context "on #{os}" do
+      let(:facts) do
+        override_facts(facts,
+                       fqdn: 'tang01.dev.lsst.org',
+                       is_virtual: true,
+                       virtual: 'kvm',
+                       dmi: {
+                         'product' => {
+                           'name' => 'KVM',
+                         },
+                       })
+      end
+      let(:node_params) do
+        {
+          role: 'tang',
+          site: 'dev',
+        }
+      end
+
+      it { is_expected.to compile.with_all_deps }
+
+      include_examples 'vm'
+      include_context 'with nm interface'
+      it { is_expected.to have_nm__connection_resource_count(1) }
+
+      context 'with enp1s0' do
+        let(:interface) { 'enp1s0' }
+
+        it_behaves_like 'nm enabled interface'
+        it_behaves_like 'nm ethernet interface'
```

Comment:
nice again, also rarely used.

Reply:
It is just an alias for `include_examples`.  I should probably be consistent and use only one of the aliases instead of both.

Reply:
although I do like the extra doc output from `it_behaves_like`...

Reply:
If you ask Ewoud about it he can probably explain the differences and when to use include_examples vs it_behaves_like, but that's beyond my knowledge :D